UnchainedLauncherGUI/UnchainedLauncherGUI.csproj

```diff
@@ -19,8 +19,9 @@
 	<Authors>Jacoby6000, AERoese (BK, DrLong), Nihilianth</Authors>
     <PackageProjectUrl>https://github.com/Chiv2-Community/UnchainedLauncher</PackageProjectUrl>
     <ApplicationIcon>assets\chiv2-unchained-logo.ico</ApplicationIcon>
-    <AssemblyVersion>1.0.0</AssemblyVersion>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
```

Comment:
Set AssemblyVersion to match git tag?
https://louisz.medium.com/use-git-tag-build-number-as-version-number-for-net-6-applications-1c1f16f3d38a

Reply:
Probably worth setting up, but it's out of scope for this PR